config/mkat_antennas.yml

```diff
@@ -0,0 +1,69 @@
+reference:
+  latitude: '-30:42:47.4'
+  longitude: '21:26:38.0'
+  altitude: 1060.0
+antennas:
+  - name=m000, diameter=13.5, north=-8.21, east=26.797, up=0.117
```

Comment:
I updated the values to be the most recent as per the delay config files.
I also added a helper script for doing the update in future, since the delay values are continually updating.